### PR TITLE
remove msdn links

### PR DESCRIPTION
**Pull request by mairaw**
Related to #2037 

Wondering if we wanna keep the links that go to previous versions

### CODE REVIEW

File: docs/framework/winforms/advanced/globalizing-windows-forms.md [hunk 1]
```diff
@@ -10,48 +11,39 @@ helpviewer_keywords:
...
+
+[Bi-Directional Support for Windows Forms Applications](bi-directional-support-for-windows-forms-applications.md)
+Explains how to create Windows-based applications that support bi-directional (right-to-left) languages.
+
+[Display of Asian Characters with the ImeMode Property](display-of-asian-characters-with-the-imemode-property.md)
+Introduces the `ImeMode` property, which is used to control the type of input a Windows Form or control accepts.
+
+## Related sections
+
+- [Globalizing and Localizing Applications](/visualstudio/ide/globalizing-and-localizing-applications)
```
Comment (rpetrusha) [inline docs/framework/winforms/advanced/globalizing-windows-forms.md]: A better link here is \[Globalizing and localizing .NET applications]\(~/docs/standard/globalization-localization.md)
The title is currently "Globalizing and Localizing .NET Framework applications, but I'll open a PR to change it.

**Review by BillWagner — APPROVED** [review]
This LGTM.

I don't know that we've decided how to handle previous versions yet.

File: docs/framework/winforms/advanced/globalizing-windows-forms.md [hunk 2]
```diff
@@ -10,48 +11,39 @@ helpviewer_keywords:
...
-  
--   [How to: Create Mirrored Windows Forms and Controls](http://msdn.microsoft.com/library/xwbz5ws0\(v=vs.110\))  
-  
--   [How to: Support Localization on Windows Forms Using AutoSize and the TableLayoutPanel Control](http://msdn.microsoft.com/library/1zkt8b33\(v=vs.110\))  
-  
--   [Walkthrough: Localizing Windows Forms](http://msdn.microsoft.com/library/y99d1cd3\(v=vs.110\))  
-  
--   [Walkthrough: Creating a Layout That Adjusts Proportion for Localization](http://msdn.microsoft.com/library/7k9fa71y\(v=vs.110\))  
-  
--   [How to: Create Message Boxes for Bi-Directional Windows Forms](http://msdn.microsoft.com/library/k1689bxh\(v=vs.110\))
```
Comment (rpetrusha) [inline docs/framework/winforms/advanced/globalizing-windows-forms.md]: I think we probably still want to include this unless/until we revise the bidirectional forms topic.

**Review by rpetrusha — APPROVED** [review]
Left a couple of comments and suggestions, @mairaw. You can decide whether you want to address them.

**Pull request merged**